Lib/idlelib/NEWS.txt

```diff
@@ -3,6 +3,9 @@ Released on 2020-10-05?
 ======================================
 
 
+bpo-27115: For Edit Go to Line, use a Query entry box subclass with
```

Comment:
"Edit Go to Line" is a bit awkward; IMO "Go to Line" is sufficient and clearer.

(Relevant to the NEWS entry as well.)

Reply:
Changed.